src/main/jbake/templates/downloads.tpl

```diff
@@ -324,6 +324,19 @@ def downloadLink(label, artifact, version, suffix) {
 	newLine()
 }
 
+def mavenDownloadLink(label, artifact, version, suffix) {
+	def path = "${artifact}/${version}/${artifact}-${version}${suffix}"
+	def base = "https://repository.apache.org/content/repositories/releases/org/apache/sling/"
```

Comment:
We need to point to Maven Central instead

> Projects must not point or refer to repository.apache.org directly in download pages, release announcements or emails. Instead, any public download links for those releases should point to Maven Central.

(https://infra.apache.org/release-distribution.html#maven)